app/src/main/res/layout/fragment_directions_form.xml

```diff
@@ -142,4 +117,105 @@
 
 	</androidx.cardview.widget.CardView>
 
+	<View
+		android:id="@+id/timeBackground"
+		android:layout_width="0dp"
+		android:layout_height="0dp"
+		android:background="?selectableItemBackgroundBorderless"
+		android:clickable="true"
+		android:focusable="true"
+		app:layout_constraintTop_toTopOf="@+id/timeIcon"
+		app:layout_constraintBottom_toBottomOf="@+id/timeIcon"
+		app:layout_constraintStart_toStartOf="@+id/timeIcon"
+		app:layout_constraintEnd_toStartOf="@id/timeBarrier"/>
+
+
+	<androidx.appcompat.widget.AppCompatImageButton
+		android:id="@+id/timeIcon"
+		android:layout_width="@dimen/buttonSize"
+		android:layout_height="@dimen/buttonSize"
+		android:layout_marginTop="8dp"
+		android:background="?selectableItemBackgroundBorderless"
+		android:clickable="false"
+		android:focusable="false"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toBottomOf="@+id/toCard"
+		app:srcCompat="@drawable/ic_time"/>
```

Comment:
Below we use `android:src` instead. I remember this being tricky and can cause crashes on Android 4. Better check what we do elsewhere.

Reply:
Good catch. I'll check it out.

Reply:
[It seems](https://android-developers.googleblog.com/2016/02/android-support-library-232.html) `app:srcCompat` should be the way to go, but also the following line should be added to `build.gradle`: `vectorDrawables.useSupportLibrary = true`. This should be done and tested in another PR though, I guess?

For now, it seems we are using both `android:src` and `app:srcCompat` without a clear distinction.

Reply:
Yeah that is the officially recommended way, but back then there were issues with it. I'd just do what is done elsewhere in Transportr for now and do any srcCompat change in a dedicated PR that needs testing on various phones especially those with minSdkVersion.

Reply:
I think that one is the last blocker for merging.

Reply:
We are currently using 23 times `app:srcCompat` and 9 times `android:src` throughout the app. So not sure what to pick here then.

Reply:
OMG! OK so let's just leave it like this and pray!